src/main/java/org/sonarqube/gradle/SonarTask.java

```diff
@@ -127,6 +125,7 @@ public void run() {
     if (isSkippedWithProperty(mapProperties)) {
       return;
     }
+    logEnvironmentInformation();
```

Comment:
I feel like this information should come up as the first information about the task. I don't think it should show up after the information about using the deprecated `sonarqube` task

Reply:
Ideally, it would be the first thing called in the task runs